index.bs

```diff
@@ -200,6 +200,13 @@ enum SpeechRecognitionMode {
     "cloud-only", // Cloud speech recognition only.
 };
 
+enum AvailabilityStatus {
+    "unavailable",
+    "downloadable",
+    "downloading",
+    "available"
+};
```

Comment:
I was thinking maybe we could reuse Translations' enum but maybe there's no point.